pkg/resolver/search.go

```diff
@@ -186,7 +188,10 @@ func (s *SearchResult) resolveUids() {
 }
 func (s *SearchResult) resolveItems() ([]map[string]interface{}, error) {
 	items := []map[string]interface{}{}
+	timer := prometheus.NewTimer(metric.DBQueryDuration.WithLabelValues("what-do-we-put"))
+	klog.Infof("xxxxxxxxxxxxxxxx [%s] ", s.query)
```

Comment:
- we could use `resolveItemsFunc` for `what-do-we-put`
- Make the log prettier and increase verbosity before we merge.
```suggestion
	timer := prometheus.NewTimer(metric.DBQueryDuration.WithLabelValues("resolveItemsFunc"))
	klog.V4).Infof("xxxxxxxxxxxxxxxx [%s] ", s.query)
```

Reply:
Both done.